app/api/webhook/login/route.ts

```diff
@@ -1,89 +1,91 @@
 import axios from "axios";
 import { NextRequest, NextResponse } from "next/server";
 
-export async function POST(req : NextRequest) {
+export async function POST(req: NextRequest) {
   const data = await req.formData();
   const content = {
     username: "MDSIG Login",
-    avatar_url: "https://cdn.discordapp.com/attachments/1222932958667870279/1222933028259496080/sig2_pfp__1.png",
+    avatar_url:
+      "https://cdn.discordapp.com/attachments/1222932958667870279/1222933028259496080/sig2_pfp__1.png",
```

Comment:
This url isn't available when I visit it. Also discord should't be used as content url.